packages/export-chart-lambda/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "package": "rm -f package.zip && yarn babel && cp -r package.json dist && cd dist && yarn --production && zip -rq ../package.zip . && cd .. && rm -rf dist",
-    "babel": "rm -rf dist && mkdir dist && ./node_modules/.bin/babel src --out-dir dist",
+    "babel": "rm -rf dist && mkdir dist && ./node_modules/.bin/babel src --out-dir dist --source-maps",
```

Comment:
We probably don't need that here since this is a Lambda function, @rohan-bes are we able to run this locally for dev/debugging purposes?